### DISCUSSION TITLE
I need to disable authorization from Postgrest completely.

### DISCUSSION THREAD

**Discussion by vjanz** [PostgREST/postgrest#2766] [Q&A] [answered]
### Environment

* PostgreSQL version: 14
* PostgREST version: latest
* Operating system: Kubernetes

### Description of the issue
I have my authorization service in Kubernetes from which I do the authorization for all the services and on that case, I want to use it for Postgrest as well. The problem is that my authorization service uses the `Authorization` header as well and now that one is interfering with the Authorization header that Postgrest has. I want to disable completely the JWT authorization that Postgrest does even when I send the Authorization header to the service.

I tried setting up some environment variables as: 

PGRST_AUTHENTICATOR=""
PGRST_AUTH_JWT=off
PGRST_JWT_SECRET_IS_BASE64=false
PGRST_JWT_SECRET=""

None of that is working and now I am receiving this response:

```
{
    "message": "JWSError JWSInvalidSignature"
}
```

Any help?

**Comment by steve-chavez** [comment, marked as answer]
> PGRST_AUTH_JWT=off

There's no such option. 

> The problem is that my authorization service uses the Authorization header as well and now that one is interfering with the Authorization header that Postgrest has

How about stripping the Authorization header from the upstream request to PostgREST? 

Then you could set the `PGRST_DB_ANON_ROLE` and just use a single role for all requests.
